SPECS-EXTENDED/slf4j/LICENSE-2.0.txt

```diff
@@ -1,202 +1,202 @@
-
```

Comment:
I have no idea what is the diff that git sees (there seems to be no actual changes), but let's fix this to not overwrite the file's history. I think something along the lines of of `git checkout main SPECS-EXTENDED/slf4j` should do the trick.

Reply:
SPECS-EXTENDED/slf4j does the trick